server/src/main/java/umm3601/Main.java

```diff
@@ -42,16 +43,21 @@ static String getEnvOrDefault(String envName, String defaultValue) {
   }
 
   /**
-   * Get the controllers for the server. You'll add your own controllers
-   * here as you create them.
+   * Get the implementations of `AddsRoutes` used for the server.
+   *
+   * These will presumably be one or more controllers, each of which
+   * implements the `AddsRoutes` interface. You'll add your own controllers
+   * in `getRouteAdders` as you create them.
    *
    * @param database The MongoDB database object used by the controllers
    *               to access the database.
-   * @return An array of controllers for the server.
+   * @return An array of implementations of `AddRoutes` for the server.
    */
-  static Controller[] getControllers(MongoDatabase database) {
-    Controller[] controllers = new Controller[] {
-      // You would add additional controllers here, as you create them.
+  static AddsRoutes[] getRouteAdders(MongoDatabase database) {
+    AddsRoutes[] controllers = new AddsRoutes[] {
+      // You would add additional controllers here, as you create them,
+      // although you need to make sure that your new controllers implement
```

Comment:
maybe change to "although you need to make sure that each of your new controllers implements"

Reply:
Good catch – thanks.